webapp/channels/src/components/advanced_text_editor/use_submit.tsx

```diff
@@ -238,70 +243,72 @@ const useSubmit = (
             channelTimezoneCount = data ? data.length : 0;
         }
 
-        if (prioritySubmitCheck(doSubmit)) {
+        if (prioritySubmitCheck(() => doSubmit(undefined, undefined, schedulingInfo))) {
```

Comment:
This is an existing bug which I'll file a ticket for, but I just noticed that any of these checks that are async and use a callback skip the following checks so, in this case, if you have a priority message, you'll never be prompted about at-mentioning too many people

Reply:
Its sounds wrong but it works correctly. Priority with bulk mention does indeed trigger the alert about at mentioning everyone because `prioritySubmitCheck` returns false unless you have set persistent notification, and if you do set persistent notification, you aren't allowed to bulk mention anywhere, so it works correctly. So everything is working correctly.

Reply:
Yeah, thats true.